### ISSUE TITLE
Quantile scale always results in the default quantize scale

### ISSUE THREAD

**Issue by wahlatlas** [floledermann/mapmap.js#38]
While I have been able to define a `threshold` scale, I have never been successful in specifying a `quantile` scale, instead it always results in the default `quantize` scale.

**Comment by floledermann** [comment]
Yes, the quantile scale is not supported in the current architecture, as it needs to be fed the actual data population to make it work, [as far as I understand](https://github.com/d3/d3-3.x-api-reference/blob/master/Quantitative-Scales.md#quantile-scales). This requires an improved architecture for scales and classification, see also #28.
